source/file/Importer.js

```diff
@@ -52,8 +52,11 @@ class Importer {
 
   /** Determine paths' absolute points, given TopoJSON delta-encoded arcs */
   _getAbsolutePaths(geometry, allArcs, transform) {
+    // flatten MultiPolygon array so that we can map the arcs to get absolute paths
+    const arcs = geometry.type === 'Polygon' ?
```

Comment:
Hmm, well as far as I can see, there's no possibly way to create a tilegram with anything other than 'Polygon', 'MultiPolygon', or 'null' geometry. All of these geometries are handled by the importer. However, I suppose we don't have any warning if someone tries to upload a non-tilegram topojson (or other unsupported file). I'll add something in there to handle that case.